### ISSUE TITLE
LoadDynamicModule: documentation for `crc` argument is wrong

### ISSUE THREAD

**Issue by fingolfin** [gap-system/gap#4235]
To quote:
```
This command loads <A>filename</A> as module. If given, the CRC checksum 
<A>crc</A> must match the value of the module (see <Ref Sect="CrcFile"/>).
```
and then it goes on with this example:
```
gap> LoadDynamicModule("./test.so");
gap> CrcFile("test.so");
2906458206
gap> LoadDynamicModule("./test.so",1);
Error, <crc> mismatch (or no support for dynamic loading) called from
<function>( <arguments> ) called from read-eval-loop
Entering break read-eval-print loop ...
you can 'quit;' to quit to outer loop, or
you can 'return;' to continue
brk> quit;
gap> LoadDynamicModule("./test.so",2906458206);
```

This suggest that the CRC of the .so file is checked. But that's wrong! Rather, the value of `crc` field in the `StructInitInfo` returned by the `Init__Dynamic` function of the module is checked.

For regular kernel extensions, this is normally always 0; the real use is for code produced by `gac`, then the CRC field is set to the CRC of the GAP source file used to generate the C code in question.

I am not even sure whether we use this `crc` test at all, though? I had a quick look and couldn't find anything that used it (or its counterpart for `LoadStaticModule`). We *do* perform CRC checks for the few files in the GAP kernel which are produced from GAP sources (`lib/oper1.g` -> `src/c_oper1.c` and `lib/type1.g` -> `src/c_type1.g`) but there the CRC check is performed by `READ_GAP_ROOT`.

In conclusion, perhaps the best way to fix the documentation is to undocumented the `crc` argument; and perhaps even remove it altogether (thus also simplifying the kernel functions `FuncLOAD_DYN` and `FuncLOAD_STAT`
